guidelines/automation-pipelines.md

```diff
@@ -0,0 +1,95 @@
+# Best Practices For Automation Pipelines<!-- omit in toc -->
```

Comment:
Why do we not want to show this in the Guidelines section? Should we add the Jekyll front-matter?

Reply:
> Why do we not want to show this in the Guidelines section? Should we add the Jekyll front-matter?

@tkuhrt A funny coincidence this one! :-) The comment means TOC as in table of contents and is a special syntax for the markdown IDE extension I use to auto-generate the **table of contents** for all my markdown documents** and is not meant to have meaning in the context of the **technical steering committee** :-)

** It has really nice features! I highly recommend it because it makes the experience of working with table of contents much closer to what you get in the well established document editing software which updates TOCs automatically as you modify the headings instead of you having to manually maintain it. It also supports auto-numbering and keeping those numbered headings up to date as well!

The relevant section of the documentation of the IDE extension:

![Screenshot from 2024-05-31 09-56-17](https://github.com/hyperledger/toc/assets/1583231/c0979af4-834e-4a51-9954-baecb5bf088b)